config/jobs/kubernetes/sig-aws/eks/presets.yaml

```diff
@@ -0,0 +1,85 @@
+    - name: AWS_K8S_TESTER_EKS_AWS_K8S_TESTER_DOWNLOAD_PATH
+      value: /tmp/aws-k8s-tester/aws-k8s-tester
+    # test mode is either "embedded" or "aws-cli" ("embedded" uses native AWS Go client, "aws-cli" will use 'aws')
+    - name: AWS_K8S_TESTER_EKS_TEST_MODE
+      value: "embedded"
+    # duration to wait before destroying EKS cluster, useful to add wait time before collecting AWS ALB access logs
+    - name: AWS_K8S_TESTER_EKS_WAIT_BEFORE_DOWN
+      value: 1m0s
+    # 'true' to destroying all AWS resources when it calls "Down"
+    - name: AWS_K8S_TESTER_EKS_DOWN
+      value: "true"
+    # 'true' to assign worker nodes across all available subnets
+    - name: AWS_K8S_TESTER_EKS_ENABLE_WORKER_NODE_HA
+      value: "true"
+    # 'true' to open port 22 in security group, and enable SSH for log dumper
+    - name: AWS_K8S_TESTER_EKS_ENABLE_NODE_SSH
+      value: "true"
+    # 'true' to enable S3 Access Logs and AWS ALB Access Logs
+    # use it for debug, dump cluster log already handles log artifacts
+    - name: AWS_K8S_TESTER_EKS_LOG_ACCESS
+      value: "false"
+    # 'true' to upload 'aws-k8s-tester' logs to S3 buckets, in addition to log dumper
+    # use it for debug, dump cluster log already handles log artifacts
+    - name: AWS_K8S_TESTER_EKS_UPLOAD_TESTER_LOGS
+      value: "false"
+    # 'true' to upload worker node logs to S3, in addition to log dumper
+    # use it for debug, dump cluster log already handles worker node logs
+    - name: AWS_K8S_TESTER_EKS_UPLOAD_WORKER_NODE_LOGS
+      value: "false"
+    # 'true' to enable debug level logs
+    - name: AWS_K8S_TESTER_EKS_LOG_DEBUG
+      value: "false"
+    # 'true' to create AWS ALB
+    - name: AWS_K8S_TESTER_EKS_ALB_ENABLE
+      value: "false"
+    # AWS test account credential mounted path, required for AWS API call
+    - name: AWS_SHARED_CREDENTIALS_FILE
+      value: /etc/eks-aws-credentials/eks-aws-credentials
+  volumeMounts:
+    - mountPath: /etc/eks-aws-credentials
+      name: eks-aws-credentials
+      readOnly: true
+  volumes:
+    - name: eks-aws-credentials
+      secret:
+        secretName: eks-aws-credentials
+
+# configuration for EKS Kubernetes e2e tests with 1.10
+- labels:
+    preset-kubernetes-e2e-aws-eks-1-10: "true"
+  env:
+    # URL to download 'kubectl', required for 'kubectl' calls to EKS
+    - name: AWS_K8S_TESTER_EKS_KUBECTL_DOWNLOAD_URL
+      value: https://amazon-eks.s3-us-west-2.amazonaws.com/1.10.3/2018-07-26/bin/linux/amd64/kubectl
+    - name: AWS_K8S_TESTER_EKS_KUBECTL_DOWNLOAD_PATH
+      value: /tmp/aws-k8s-tester/kubectl
+    # URL to download 'aws-iam-authenticator', required for 'kubectl' calls to EKS
+    - name: AWS_K8S_TESTER_EKS_AWS_IAM_AUTHENTICATOR_DOWNLOAD_URL
+      value: https://amazon-eks.s3-us-west-2.amazonaws.com/1.10.3/2018-07-26/bin/linux/amd64/aws-iam-authenticator
+    - name: AWS_K8S_TESTER_EKS_AWS_IAM_AUTHENTICATOR_DOWNLOAD_PATH
+      value: /tmp/aws-k8s-tester/aws-iam-authenticator
```

Comment:
Same as above.

Reply:
Also for other ones below which aren't obvious to me why they're version specific.